### PR TITLE
Fixes arrow head, thickness and color in my-plumb directive

### PR DESCRIPTION
**Pull request by ajainarayanan**
Fixes the arrow styles (thickness, color and arrow head)

### CODE REVIEW

**Comment by astewes** [comment]
LGTM

File: cdap-ui/app/directives/plumb/my-plumb-factory.js [hunk 1]
```diff
@@ -16,11 +16,11 @@ angular.module(PKG.name + '.commons')
       maxConnections: -1,
       paintStyle: {
         strokeStyle: 'white',
-        fillStyle: '#666',
+        fillStyle: '#666e82',
```
Comment (elfenheart) [inline cdap-ui/app/directives/plumb/my-plumb-factory.js]: Is it possible to use variable already defined in variable.less?

Comment (ajainarayanan) [inline cdap-ui/app/directives/plumb/my-plumb-factory.js]: I don't believe we can use the variable defined in a less file in javascript file. We don't ship less files right.. Any suggestions to do it in a straight forward way apart from querying through css selectors ?

Comment (elfenheart) [inline cdap-ui/app/directives/plumb/my-plumb-factory.js]: lol, my bad...didn't notice it's in JS :grin:

**Comment by elfenheart** [comment]
:+1: LGTM

**Pull request merged**